.github/workflows/workflow.yaml

```diff
@@ -96,29 +98,35 @@ jobs:
           cp .bazelrc.local rules_haskell_nix
           cp .bazelrc.local rules_haskell_tests
       - name: Check Bazel version
-        run: |
-          nix-shell --arg docTools false --argstr ghcVersion ${{ matrix.ghc }} --pure --run .ci/check-bazel-version
+        uses: tweag/run-nix-shell@v0
+        with:
+          options: ${{ env.NIX_SHELL_ARGS }}
+          run: .ci/check-bazel-version
       - name: Build & test - rules_haskell
         if: matrix.module == 'rules_haskell'
-        run: |
-          nix-shell --arg docTools false --argstr ghcVersion ${{ matrix.ghc }} --pure --run '
+        uses: tweag/run-nix-shell@v0
+        with:
+          options: ${{ env.NIX_SHELL_ARGS }}
+          run: |
             set -euo pipefail
             bazel test //...
             bazel build //docs:api_html
             bazel build //docs:guide_html
-            '
       - name: Build & test - rules_haskell_nix
         if: matrix.module == 'rules_haskell_nix'
-        run: |
-          nix-shell --arg docTools false --argstr ghcVersion ${{ matrix.ghc }} --pure --run '
+        uses: tweag/run-nix-shell@v0
+        with:
+          options: ${{ env.NIX_SHELL_ARGS }}
+          run: |
             set -euo pipefail
             cd rules_haskell_nix
```

Comment:
Could we use `working-directory:` instead and drop the `cd command` and `set -euo pipefail`?

Reply:
It looks like I need to add an attribute to `run-nix-shell` to specify the location of `shell.nix`. I just added [an issue](https://github.com/tweag/run-nix-shell/issues/4) to track. Are you OK with me leaving the `cd ...` for now?